test/ssc_test/cmod_battery_pvsamv1_test.cpp

```diff
@@ -345,14 +345,14 @@ TEST_F(CMPvsamv1BatteryIntegration_cmod_pvsamv1, PPA_ACBatteryModelIntegration)
     grid_and_rate_defaults(data);
     singleowner_defaults(data);
 
-    ssc_number_t expectedEnergy[3] = { 37307927, 37277087, 37277087 };
-    ssc_number_t expectedBatteryChargeEnergy[3] = { 14779, 315293, 315293 }; // No rate model means battery use is low
-    ssc_number_t expectedBatteryDischargeEnergy[3] = { 14663, 284336, 284336 };
+    ssc_number_t expectedEnergy[3] = { 37307927, 37306925, 37308021 };
+    ssc_number_t expectedBatteryChargeEnergy[3] = { 14779, 25275, 14779 }; // No rate model means battery use is low
+    ssc_number_t expectedBatteryDischargeEnergy[3] = { 14663, 24062, 14663 };
 
     ssc_number_t peakKwCharge[3] = { -1040.2, -1051.5, -1051.5 };
     ssc_number_t peakKwDischarge[3] = { 967.5, 969.5, 969.5 };
     ssc_number_t peakCycles[3] = { 1, 1, 1 };
-    ssc_number_t avgCycles[3] = { 0.003, 0.2356, 0.2356 };
+    ssc_number_t avgCycles[3] = { 0.003, 0.014, 0.003 };
```

Comment:
What led to these large changes?

Reply:
This test has grid charging enabled. The root error for these changes was that the code was enabling retail rates when it shouldn't have been. So the changes to cmod_battery increased the price of charging for this test, which decreased the cycles in years 2 & 3. I think the PPA rate escalation was the reason for the large jump in cycles prior to this change, but now the charging costs escalate at the same rate as the discharging revenue.

Reply:
Ok, thanks!